tests/core.py

```diff
@@ -50,6 +52,31 @@ def setUp(self):
         self.dag_bash = self.dagbag.dags['example_bash_operator']
         self.runme_0 = self.dag_bash.get_task('runme_0')
 
+    def test_schedule_dag_no_previous_runs(self):
+        dag = DAG(TEST_DAG_ID+'test_schedule_dag_no_previous_runs') 
+        dag.tasks = [models.BaseOperator(task_id="faketastic", owner='Also fake',
+            start_date=datetime(2015, 1, 2, 0, 0))]
+        dag_run = jobs.SchedulerJob(test_mode=True).schedule_dag(dag)
+        assert dag_run is not None
+        assert dag_run.dag_id == dag.dag_id
+        assert dag_run.run_id is not None
+        assert dag_run.run_id != ''
+        assert dag_run.execution_date == datetime(2015, 1, 2, 0, 0), (
+                'dag_run.execution_date did not match expectation: %s' % dag_run.execution_date)        
```

Comment:
@mistercrunch I'll have these fixed in the next PR in the series, in terms of using `.format`